src/main/kotlin/no/nav/faktureringskomponenten/domain/models/FakturaLinje.kt

```diff
@@ -34,7 +34,33 @@ class FakturaLinje(
     @Column(name = "belop", nullable = false)
     val belop: BigDecimal = BigDecimal(0),
 ) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as FakturaLinje
+
+        if (periodeFra != other.periodeFra) return false
+        if (periodeTil != other.periodeTil) return false
+        if (beskrivelse != other.beskrivelse) return false
+        if (antall != other.antall) return false
+        if (enhetsprisPerManed != other.enhetsprisPerManed) return false
+        if (belop != other.belop) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
```

Comment:
Er det mulig å bare bruke ett standard bibliotek her? F.eks
    return Objects.hash(periodeFra, periodeTil, beskrivelse, antall, enhetsprisPerManed, belop)

Reply:
Jeg har brukt autogenerering med IntelliJ som ikke har andre støtte.

Jeg kan bruke `Objects.hash` som ser penere ut og gjør akkurat det samme med en array, men tenker ikke at det er viktig nok.

Det finnes tilsynelatende ikke en bedre måte å gjøre dette i ren Kotlin ref. https://discuss.kotlinlang.org/t/best-replacement-for-java-objects-hash/17933/2

Hva synes du?

Reply:
Selv ville jeg brukt Objects.hash, kun pga att jeg er fan av å bruke eksisterende biblioteker. 
Men som du sier selv, det er ikke så nøye, og så får vi jo faktisk en kotlin implementasjon her.